mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -157,7 +166,15 @@ def playlistclear(context, name):
 
         Clears the playlist ``NAME.m3u``.
     """
-    raise exceptions.MpdNotImplemented  # TODO
+    uri = context.lookup_playlist_uri_from_name(name)
+    playlist = uri is not None and context.core.playlists.lookup(uri).get()
+    if not playlist:
```

Comment:
Test for it is already there - https://github.com/mopidy/mopidy/pull/1187/files#diff-f47c92bf0379f3f53b0dd6985c54eaf9R233 - but I added comment.

And for 1st question, if I understood correctly, `context.core.playlists.create` should create playlist using default backend, if no scheme is specified. And there is probably no better way, I can't guess anything from empty playlist and MPD client can't tell the difference.